src/plugins/auto/src/plugin.cpp

```diff
@@ -549,11 +561,19 @@ std::list<DeviceInformation> MultiDeviceInferencePlugin::GetValidDevice(
             continue;
         }
         if (item.deviceName.find("GPU") == 0) {
-            auto& gpuUniqueName = item.uniqueName;
-            if (gpuUniqueName.find("iGPU") != std::string::npos) {
-                iGPU.push_back(item);
-            } else if (gpuUniqueName.find("dGPU") != std::string::npos) {
-                dGPU.push_back(item);
+            auto deviceArchitecture = GetCore()->GetMetric(item.deviceName, METRIC_KEY(DEVICE_ARCHITECTURE)).as<std::string>();
+            if (isIntelDevice(deviceArchitecture)) {
```

Comment:
Does this logic mean we just only accept GPU from INTEL? Is this our expected behavior?

Reply:
From the discussion in cvs-106628, will not add other GPU to device priorities. Please check this @peterchen-intel @wangleis

Reply:
Let's select Intel dGPU as first priority and keep other dGPU as low priority which just higher than CPU.